src/services/media-service.js

```diff
@@ -22,6 +22,14 @@ export async function getMediaFiles(projectId) {
   return rows
 }
 
+export async function getOneTimeMediaFiles(projectId) {
+  const [rows] = await sequelizeConn.query(
+    `SELECT * FROM media_files WHERE project_id = ? AND is_copyrighted > 0 AND copyright_license = 8`,
```

Comment:
The "*" gets all the data in the media_files table and returns it to the client. This is not ideal since it contains A LOT of data and exposes sensitive information. Since we only need the media_id here, let's only include that in the selection.